src/styles/dotcms-theme/components/_dynamicdialog.scss

```diff
@@ -0,0 +1,8 @@
+@use "variables" as *;
+
+p-dynamicdialog {
+    .p-component-overlay {
+        align-items: baseline;
+        padding: 7rem;
```

Comment:
$spacing?

Reply:
Don't have a 7rem spacing because is very uncommon.

Reply:
@fmontes `padding: $spacing-6 + $spacing-9 ; // 7rem`

Reply:
Do we want to go there? because is not related to that contextually.

Reply:
Padding is used to space out elements, so technically it is a spacing.